diagnostic.md

```diff
@@ -48,11 +50,23 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    You would create/add to a model for movies.  Something like:
+
+    import Ember from 'ember';
+
+    export default Ember.Route.extend({
+      model: function(){
+        return [
+          {
+          id: 123
+          },
+        ]
+      }
+    });
     ```
```

Comment:
So actually here, we'd want to take advantage of the fact that a Route model can accept a parameter of `params` that gives us access to all route params. Thus, in the model, we'd access the value 123 coming from the route params with `params.movie_id`